File: Baekjoon/python/31090.py
# source : https://www.acmicpc.net/problem/31090

import sys
input = sys.stdin.readline

def main():
    # input
    for _ in range(int(input())):
        # input
        year = input()
        
        print('Good' if not (int(year) + 1) % int(year[2:4]) else 'Bye')

if __name__ == '__main__':
    main()

File: Baekjoon/nodejs/31090.js
// source : https://www.acmicpc.net/problem/31090

const input = require('fs').readFileSync('example.txt').toString().trim().split('\n');

const years = input.slice(1);

for (year of years) {
    console.log(!((Number(year) + 1) % Number(year.substr(2, 2))) ? 'Good' : 'Bye');
}